srcpkgs/ssh-tools/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'ssh-tools'
+pkgname=ssh-tools
+version=1.6
+revision=1
+makedepends="help2man"
```

Comment:
hostmakedepends

Reply:
adding hostmakedepends or replacing makedepends with hostmakedepends?

Reply:
help2man should be hostmakedepends

Reply:
help2man is a problem for cross build, we usually omit it everywhere, as help is already there

Reply:
So I can drop makedepends and the manpage build steps completely?

Reply:
Since you are calling it manually in this case, definitely.

Reply:
done